lib/shelley/bench/Restore.hs

```diff
@@ -287,6 +287,9 @@ cardanoRestoreBench tr c socketFile = do
           -- to-100% time.
           benchRestoreMultipleWallets 1 (unsafeMkPercentage 0.1)
         , benchRestoreMultipleWallets 10 (unsafeMkPercentage 0.01)
+        , benchRestoreMultipleWallets 20 (unsafeMkPercentage 0.01)
+        , benchRestoreMultipleWallets 40 (unsafeMkPercentage 0.01)
+        , benchRestoreMultipleWallets 80 (unsafeMkPercentage 0.01)
         , benchRestoreMultipleWallets 100 (unsafeMkPercentage 0.01)
```

Comment:
What about removing 100, so we keep a nice x2 factor between each bench.

Reply:
Or make it 160? :smile_cat:  (Perhaps that would be **_too_** long.)

Reply:
By keeping 100 we can still reason about 1 vs 10 vs 100.